scripts/ci/in_container/kubernetes/docker/rebuild_airflow_image.sh

```diff
@@ -38,46 +38,25 @@ cp /entrypoint.sh scripts/docker/
 echo
 echo "Building image from ${AIRFLOW_CI_IMAGE} with latest sources"
 echo
-start_output_heartbeat "Rebuilding Kubernetes image" 3
-docker build \
-    --build-arg PYTHON_BASE_IMAGE="${PYTHON_BASE_IMAGE}" \
-    --build-arg PYTHON_MAJOR_MINOR_VERSION="${PYTHON_MAJOR_MINOR_VERSION}" \
-    --build-arg AIRFLOW_VERSION="${AIRFLOW_VERSION}" \
-    --build-arg AIRFLOW_EXTRAS="${AIRFLOW_EXTRAS}" \
-    --build-arg ADDITIONAL_AIRFLOW_EXTRAS="${ADDITIONAL_AIRFLOW_EXTRAS}" \
-    --build-arg ADDITIONAL_PYTHON_DEPS="${ADDITIONAL_PYTHON_DEPS}" \
-    --build-arg AIRFLOW_BRANCH="${AIRFLOW_BRANCH}" \
-    --build-arg AIRFLOW_CONTAINER_CI_OPTIMISED_BUILD="${AIRFLOW_CONTAINER_CI_OPTIMISED_BUILD}" \
-    --build-arg UPGRADE_TO_LATEST_REQUIREMENTS="${UPGRADE_TO_LATEST_REQUIREMENTS}" \
-    --build-arg HOME="${HOME}" \
-    --cache-from "${AIRFLOW_CI_IMAGE}" \
-    --tag="${AIRFLOW_CI_IMAGE}" \
-    --target="main" \
-    -f Dockerfile.ci . >> "${OUTPUT_LOG}"
-echo
+#export AIRFLOW_PROD_BASE_TAG="${DEFAULT_BRANCH}-python${PYTHON_MAJOR_MINOR_VERSION}"
```

Comment:
@potiuk can you please point me to how I can initialize these env variables. I can't find it and there are a lot of bash scripts to navigate.

Reply:
Those variablse are all set in the HOST and you need to pass them to the container. But I think this is not really needed . I just rebased #8265  which moves setting up kubernetes kind cluster and deploying production image to host rather than to CI image and I am going to finish it up this week, so  I think it's better to do it this way.

Reply:
@potiuk I agree your way is better, but until this is merged the k8s won't pass because the image is so massive so we should figure out a hack for now